ProductClass.js

```diff
@@ -0,0 +1,26 @@
+export class PRODUCT {
+  constructor(name, description, price, tags, images) {
+    this.name = name;
+    this.description = description;
+    this.price = price;
+    this.tags = tags;
+    this.images = images;
+    this.찜꽁 = "💖x 0"; //찜꽁
+  }
+  favorite() {
+    console.log(`${this.name}을(를) 찜 하였습니다.💖`);
+
+    let num = Number(this.찜꽁[3]);
+    num += 1;
+    this.찜꽁 = this.찜꽁.slice(0, 3) + num;
+  }
+}
+
+const Manufacturer = ["IG", "SamsSong", "P.APPLE", "Soni", "Phylaps"];
+
+export class ElectronicProduct extends PRODUCT {
+  constructor(name, description, price, tags, images, 찜꽁) {
+    super(name, description, price, tags, images, 찜꽁);
+    this.manufacturer = Manufacturer[Math.floor(Math.random() * 5)]; //'Math.floor(Math.random() * 5)' <= mdn 참고
+  }
```

Comment:
부모 클래스인 PRODUCT의 생성자는 5개 파라미터만 받는데, super()에 6개를 전달하고 있습니다. 
이 `찜꽁` 파라미터는 무시될 거예요.
아래 방식으로 개선해보면 좋을 것 같아요.
```suggestion
export class ElectronicProduct extends PRODUCT {
  constructor(name, description, price, tags, images) {
    super(name, description, price, tags, images);
    this.manufacturer = Manufacturer[Math.floor(Math.random() * Manufacturer.length)];
  }
}
```